packages/flutter_test/lib/src/window.dart

```diff
@@ -503,6 +521,85 @@ class TestPlatformDispatcher implements PlatformDispatcher {
     extraKeys.forEach(_testViews.remove);
   }
 
+  @override
+  ErrorCallback? get onError => _platformDispatcher.onError;
+  @override
+  set onError(ErrorCallback? value) {
+    _platformDispatcher.onError;
+  }
+
+  @override
+  VoidCallback? get onSystemFontFamilyChanged => _platformDispatcher.onSystemFontFamilyChanged;
```

Comment:
(not for this PR): We need to enforce somehow that either all these values are reset at the end of the test or auto-reset them before the next test runs. Otherwise, they'll leak into the next test.

Reply:
Yeah... I've got a lot of concerns with these callbacks and test states. The fact that the binding exists between tests is concerning to me in general, so add in the ability to add random callbacks and I can see a _lot_ of potential issues with them.

Reply:
Added it as part of the requirements for #121917.